packages/cli/src/util/command.ts

```diff
@@ -8,7 +8,13 @@ export function createMedplumCommand(name: string): Command {
     .option('--token-url <tokenUrl>', 'FHIR server token url')
     .option('--authorize-url <authorizeUrl>', 'FHIR server authorize url')
     .option('--fhir-url-path <fhirUrlPath>', 'FHIR server url path')
-    .option('-p, --profile <profile>', 'Profile name')
+    .option('--scope <scope>', 'JWT scope')
+    .option('--access-token <accessToken>', 'Access token for token exchange authentication')
+    .option('--callback-url <callbackUrl>', 'Callback URL for authorization code flow')
+    .option('--subject <subject>', 'Subject for JWT authentication')
+    .option('--audience <audience>', 'Audience for JWT authentication')
+    .option('--issuer, <issuer>', 'Issuer for JWT authentication')
+    .option('-p, --profile <name>', 'Profile name')
     .addOption(
       new Option('--auth-type <authType>', 'Type of authentication').choices([
```

Comment:
Perhaps change this from 'jwt-bearer' to 'health-gorilla'?

Reply:
I'd like to avoid that unless we're absolutely convinced that this only works for health gorilla.